benchmarks/Sources/VerifierBenchmark.swift

```diff
@@ -0,0 +1,590 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftCertificates open source project
+//
+// Copyright (c) 2023 Apple Inc. and the SwiftCertificates project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftCertificates project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import X509
+import Foundation
+import Crypto
+import SwiftASN1
+
+public func verifier() async {
```

Comment:
I would love if we separate each benchmark into a separate file and move the supporting code into separate files as well. I don't wanna end up with a massive file again like the current NIO benchmarks have.

Reply:
It is a single benchmark. Not sure if we want separate benchmarks per validation use case. Maybe successful and unsuccessful as separate benchmarks but that's it.